starrocks/benchmark.sh

```diff
@@ -39,7 +40,7 @@ sleep 30
 # Prepare Data
 cd ../
 wget --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz'
-gzip -d hits.tsv.gz
+gzip -f -d hits.tsv.gz
```

Comment:
Was there a problem with the previous gzip call which is fixed by force (`-f`)?

Reply:
When running in a shell it will ask for confirmation if file already exists.

Reply:
Allow me to revert this and make a separate PR which adds `-f` to _all_ usage of gzip in ClickBench.

Reply:
https://github.com/ClickHouse/ClickBench/pull/358